plugins/janus_streaming.c

```diff
@@ -721,6 +729,11 @@ rtspiface = network interface IP address or device name to listen on when receiv
 #include "../utils.h"
 #include "../ip-utils.h"
 
+/* Default settings */
+#define JANUS_STREAMING_DEFAULT_SESSION_TIMEOUT 0 /* Overwrite the RTSP session timeout. If set to zero, the RTSP timeout is derived from a session. */
+#define JANUS_STREAMING_DEFAULT_RECONNECT_DELAY 5 /* Reconnecting delay in seconds. */
+#define JANUS_STREAMING_DEFAULT_CURL_TIMEOUT 10L /* Communication timeout for cURL. */
+#define JANUS_STREAMING_DEFAULT_CURL_CONNECT_TIMEOUT 5L /* Conection timeout for cURL. */
```

Comment:
nit: `Connection timeout`

Reply:
Thank you so much for proofreading my English. :)